ingest/workflow/snakemake_rules/trigger_rebuild.smk

```diff
@@ -15,5 +15,5 @@ rule trigger_build:
         touch("data/trigger/rebuild.done")
     shell:
         """
-        ./bin/trigger-on-new-data {input.metadata_upload} {input.fasta_upload}
+        ./vendored/trigger-on-new-data nextstrain/monkeypox rebuild {input.metadata_upload} {input.fasta_upload}
```

Comment:
Ah, this made me realize `trigger-on-new-data` would need the repo name (without org) since it [calls on the centralized trigger](https://github.com/nextstrain/ingest/blob/5d908187d13cbce27253af5163b4803d6bac03a6/trigger-on-new-data#L22) that [only uses the repo name](https://github.com/nextstrain/ingest/blob/5d908187d13cbce27253af5163b4803d6bac03a6/trigger#L45).

I think it would be preferable to change `trigger` to accept org/repo to match our `notify-*` scripts?

Reply:
Oh good catch, changing to org/repo for consistency makes sense! Are you able to do that in nextstrain/ingest? I can update this PR once that's done.

Reply:
Ah, thanks for updating nextstrain/ingest!

Reply:
Done in [force-pushes](https://github.com/nextstrain/monkeypox/compare/bf3c93ae25486de943f512fd9b9ef309059168bd..02d6119a73b1faacbe0879456f47c7ef18a33edf).